Source/Core/DiscIO/VolumeCreator.cpp

```diff
@@ -128,7 +128,7 @@ static std::unique_ptr<IVolume> CreateVolumeFromCryptedWiiImage(std::unique_ptr<
 
 	struct SPartition
 	{
-		SPartition(u64 offset, u32 type) : offset(offset), type(type) {}
+		SPartition(u64 _offset, u32 _type) : offset(_offset), type(_type) {}
```

Comment:
http://stackoverflow.com/questions/228783/what-are-the-rules-about-using-an-underscore-in-a-c-identifier says "All identifiers that begin with an underscore are always reserved for use as identifiers with file scope in both the ordinary and tag name spaces.", which seems to imply we can shadow them. (We can't have macros with names like that, though.)

Personally I'd keep it like that, to match Python, but your choice.